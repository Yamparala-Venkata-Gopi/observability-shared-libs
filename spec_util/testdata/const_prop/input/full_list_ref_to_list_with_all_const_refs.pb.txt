# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    ref: {
      method_index: 2
      arg_ref: {
        key: "list_nested_const_ref_arg"
        data_ref: {
          list_ref: {
            full_list: {}
          }
        }
      }
    }
  }
}
