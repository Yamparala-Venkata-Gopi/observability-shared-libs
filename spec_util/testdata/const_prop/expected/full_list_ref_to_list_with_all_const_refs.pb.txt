# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    value: {
      # The full list is a constant.
      list: {
        elems: {
          primitive: {
            type_hint: "string_constant_1"
            string_value: {
              value: "7c73565e-0d62-49b0-84e7-3063f4d53d9c"
            }
          }
        }
      }
    }
  }
}
